insights/components/cloud_provider.py

```diff
@@ -0,0 +1,59 @@
+"""
+Components identify Cloud Provider
+===================================
+
+The ``Is*`` component is valid if the
+:py:class:`insights.combiners.cloud_provider.CloudProvider` combiner indicates
+the host is from the specific Cloud Provider.  Otherwise, it raises a
+:py:class:`insights.core.dr.SkipComponent` to prevent dependent components from
+executing.
+
+"""
+from insights.core.dr import SkipComponent
+from insights.core.plugins import component
+from insights.combiners.cloud_provider import CloudProvider
+
+
+@component(CloudProvider)
+class IsAWS(object):
```

Comment:
@xiangce you'll need to add these to the default manifest in `insights/collect.py` in order for these components to load during collection.  When you look there you'll see how the IsRhel components were added.

Reply:
Added. @bfahr Thanks!